plugins/source/shopify/docs/tables/shopify_abandoned_checkouts.md

```diff
@@ -1,6 +1,7 @@
 # Table: shopify_abandoned_checkouts
 
 The primary key for this table is **id**.
+It supports incremental syncs.
```

Comment:
It's optional, but you can also use the `IncrementalKey` column creation option to indicate which column(s) are used as the cursor, and this will then get mentioned in the docs here as well.

Reply:
I'm talking about `IncrementalKey`, a column-level option, which is different from `IsIncremental`, a table-level option.

Reply:
For more context: `IsIncremental` is required, as `overwrite-delete-stale` relies on it to know not to delete stale entries. Column-level `IncrementalKey` is optional and only for documentation purposes right now.

Reply:
Thanks, added